### PR TITLE
Add Vagrantfile for testing

### PR DESCRIPTION
**Pull request by PVince81**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+## Testing using Vagrant
+
+Vagrant makes it possible to run Ansible scripts inside a virtual machine (backed by VirtualBox).
+
+- For the first time setup, please run `vagrant up`.
+- Then after making changes to the playbooks, roles or inventory, please run `vagrant provision` to apply the changes to the virtual machine.
+- The deployed ownCloud instance is accessible under http://172.30.1.2/ with user "admin" and password "owncloud".
+- The command `vagrant ssh` makes it possible to enter the VM for debugging purposes.
+
```
Comment (xoxys) [inline README.md]: Please move the testing documentation to `_docs/testing.md`. Please be aware that this file is a hugo page and need also a front matter. See https://raw.githubusercontent.com/owncloud-ansible/playground/master/_docs/setup.md

File: inventories/vagrant/group_vars/owncloud.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+owncloud_version: "10.3.1"
+owncloud_fqdn: vagrant.owncloud.demo
+
+# If you would like to access your ownCloud by IP
+# you can add it to trusted domains.
+owncloud_trusted_domains:
+  - "{{ owncloud_fqdn }}"
+  - "{{ ansible_default_ipv4.address }}"
+  - "172.30.1.2"
```
Comment (xoxys) [inline inventories/vagrant/group_vars/owncloud.yml:10]: Is this IP assigned inside the Vagrant VM? If so, its maybe duplicated to `"{{ ansible_default_ipv4.address }}"`

**Review by xoxys — CHANGES_REQUESTED** [review]

File: inventories/vagrant/group_vars/owncloud.yml (hunk 2 above)
Comment (PVince81) [reply inventories/vagrant/group_vars/owncloud.yml:10]: it is assigned in Vagrantfile manually (hard-coded).

I noticed that `{{ ansible_default_ipv4.address }}` was returning a completely different IP which is not usable

Comment (xoxys) [reply inventories/vagrant/group_vars/owncloud.yml:10]: Ok, keep it for now.

**Comment by PVince81** [comment]
@xoxys I've moved the doc as requested, see last commit.

File: _docs/testing.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Testing
+---
+
+# Testing
```
Comment (xoxys) [inline _docs/testing.md]: Please remove it. It's duplicate otherwise because h1 is injected from `title` front matter attribute

**Review by xoxys — CHANGES_REQUESTED** [review]

File: _docs/testing.md (hunk 3 above)
Comment (PVince81) [reply _docs/testing.md]: I was also wondering about this but haven't been able to quickly find the answer or examples online.

I'll remove it

**Review by xoxys — APPROVED** [review]

**Comment by xoxys** [comment]
Thanks

**Pull request merged**